README.md

```diff
@@ -93,4 +101,4 @@ the License.
 [dev-preview-download]: https://dl.google.com/dl/androidthings/rpi3/devpreview/3.1/androidthings_rpi3_devpreview_3_1.zip
 [set-activity-controls]: https://developers.google.com/assistant/sdk/prototype/getting-started-other-platforms/config-dev-project-and-account#set-activity-controls
 [mic]: https://www.adafruit.com/product/3367
-[speaker]: https://www.adafruit.com/product/3369
\ No newline at end of file
+[speaker]: https://www.adafruit.com/product/3369
```

Comment:
looks like github automatically added or removed the new line at the end of the file when I edited inline in the browser